### DISCUSSION TITLE
Table row highlight/Selection mode

### DISCUSSION THREAD

**Discussion by ThiyaDezine** [dxc-technology/halstack-react#1670] [Ideas] [answered]
To enhance the table component experience, we can highlight the table row(Hover and selection). 
![Screenshot (521)](https://github.com/dxc-technology/halstack-react/assets/122968305/d57ca514-8eef-4ac3-b7be-42a76f9561b1)

**Comment by GomezIvann** [comment]
From what I understood, these are the two functionalities requested:

1. **Clickable or interactive rows** (maybe to go to another page or display more detailed information about the selected row data).
2. **Row multi-selection** (checkbox/radio button).

**Reply by raquelarrojo** [reply]
I agree with Iván. So when there is one these two functionalities, we should add the hover/selected states. And I think there should be also the simple table without any selection (and any hover/selected states).

**Reply by rconcepcion3** [reply]
Additional consideration here is a hover mouse icon change (similar to links) to further indicate a selectable row. So to summarize,we're looking at three table variants:

- Simple with no interactions
- Clickable/interactive rows (with hover indications and active state)
- Multi-selectable rows (checkbox plus selected state color)

**Reply by kukoc** [reply, marked as answer]
To be included as requirement when redesigning the Table component.
